addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties

```diff
@@ -24,6 +25,17 @@ alertFilters.dialog.add.field.label.attack		= Attack:
 alertFilters.dialog.add.field.label.attackregex	= Attack is Regex?
 alertFilters.dialog.add.field.label.evidence	= Evidence: 
 alertFilters.dialog.add.field.label.evidenceregex	= Evidence is Regex?
+alertFilters.dialog.add.field.label.test		= Test Filter:
+alertFilters.dialog.add.field.label.apply		= Apply Filter:
+
+alertFilters.dialog.add.label.scope.global		= Global
+
+alertFilters.dialog.button.apply	= Apply
+alertFilters.dialog.button.test		= Test
+alertFilters.dialog.filter.state.notapplied = Not applied
+alertFilters.dialog.filter.state.nottested = Not tested
+alertFilters.dialog.filter.state.appliesto = Applies to {0} alerts
+alertFilters.dialog.filter.state.appliedto = Applied to {0} alerts
```

Comment:
Duplicate

Reply:
Its not actually - "Applies" vs "Applied" :)

Reply:
OMG! Yikes... Okay